recipes-debian/neard/neard_debian.bb

```diff
@@ -0,0 +1,57 @@
+# base recipe:
```

Comment:
@cheat-sc would you write up headers please.

Reply:
I'm sorry. I have written headers.

Reply:
thanks!